### DISCUSSION TITLE
How to keep the background and only change the object been segmented?

### DISCUSSION THREAD

**Discussion by imadcat** [lllyasviel/ControlNet#197] [Q&A]
When I used the semantic segmentation model to change the specific object in the foreground, I would like to keep the background pixels completely unchanged. 
Is there any prompt tricks to achieve that? Or it is not supported by the current algorithm?

![image](https://user-images.githubusercontent.com/1559729/222059248-ead7189a-b117-487d-b04b-9206a3f8a584.png)

Note this is almost the opposite question to this one: https://github.com/lllyasviel/ControlNet/discussions/123#discussion-4879552

**Comment by geroldmeisinger** [comment]
Inpainting or Photoshop

**Reply by imadcat** [reply]
Yes I know the concept of inpainting, the question is whether ControlNet is capable of doing so?

**Reply by icemusike** [reply]
I have the same question, how to only change background but keep model in place.

**Reply by geroldmeisinger** [reply]
you can use inpainting with control net

**Reply by nighting0le01** [reply]
can we use 4 channel unet + inpainting to do so?
